tests/spread/tutorials/basic/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: smoke test for Chisel
```

Comment:
basic.. smoke test.. etc.. there's no hint about what is in the test other than reading it.  What is the actual summary of what is being done below?

Reply:
Should be a bit more explicit now